templates/express_oauth_microsoft/index.js

```diff
@@ -0,0 +1,60 @@
+import { configDotenv } from 'dotenv'
+configDotenv({ path: "./templates/express_oauth_microsoft/.env" }) // Use when testing the tool
+// configDotenv({ path: "./.env" }) // use when testing the template
+

```

Comment:
not necessary. comment this one out. uncomment the below line. ideally when the template is generated, it should work.